src/Promitor.Agents.Scraper/Validation/MetricDefinitions/ResourceTypes/EventHubsMetricValidator.cs

```diff
@@ -18,8 +17,12 @@ public IEnumerable<string> Validate(MetricDefinition metricDefinition)
 
             var errorMessages = new List<string>();
 
-            var configuredDimension = metricDefinition.AzureMetricConfiguration?.Dimension?.Name;
-            var isEntityNameDimensionConfigured = string.IsNullOrWhiteSpace(configuredDimension) == false && configuredDimension.Equals(EntityNameDimension, StringComparison.InvariantCultureIgnoreCase);
+            if (metricDefinition.AzureMetricConfiguration?.Dimensions?.Count > 1)
```

Comment:
I assume you meant to call `Any()` on `Dimensions`? Wouldn’t that mean that the error message is added when `Dimensions` is empty, instead of when there are too many metric dimensions?

Reply:
Yes 😅 

Good point, resolving.